tarun_kumar_PostgreAssignment/src/controller/mark-controller.ts

```diff
@@ -0,0 +1,42 @@
+import { Context } from 'vm';
+import * as subjectService from '../services/subject-service';
+import * as markService from '../services/mark-service';
+
+
+export async function addMarks(ctx: Context) {
+    try {
+        const requestData = ctx.request.body;
+
+        const student_id: string = requestData.student_id;
+        const subject_name: string = requestData.subject_name.toLowerCase();
+        const marks: number = requestData.marks;
+        const subject_id: string = await subjectService.getSubjectId(subject_name);
+
+        await markService.addMark(student_id, subject_id, marks);
```

Comment:
can we send the request body it self as an object instead of sending the properties one by one.